### PR TITLE
Add roles on cap task

### PR DESCRIPTION
**Pull request by horaciob**

### CODE REVIEW

File: lib/capistrano/tasks/thin.cap [hunk 1]
```diff
@@ -4,7 +4,7 @@ namespace :thin do
   commands.each do |command|
     desc "thin #{command}"
     task command do
-      on roles(:app), in: :sequence, wait: 5 do
+      on roles(fetch(:thin_roles, :app), in: :sequence, wait: 5 do
```
Comment (nirnaeth) [inline lib/capistrano/tasks/thin.cap]: isn't a closing parenthesis missing?

Comment (horaciob) [inline lib/capistrano/tasks/thin.cap]: ups! ;)

**Review by horaciob — COMMENTED** [review]
Done

**Comment by alepore** [comment]
Thanks!

**Pull request merged**